src/main/java/org/jabref/gui/sidepane/SidePane.java

```diff
@@ -53,7 +54,21 @@ public SidePane(PreferencesService preferencesService,
         updateView();
     }
 
+    private void initComponents() {
+        components.put(SidePaneType.GROUPS, new GroupSidePane(this, taskExecutor, stateManager, preferencesService, dialogService));
```

Comment:
For better performance, it would be good if not all side panes are initalized on startup, but only if they are really used. For example, some users don't use open office and thus never open this sidepane.

Reply:
I'll note that when implementing `SidePaneContainterView` the replacement for `SidePane`, for now I'll ignore lazy loading until the final touches, I'll make sure to wrap the logic of providing side pane instances into one method so the change will be easy.